ipaserver/install/ca.py

```diff
@@ -317,15 +316,15 @@ def install_step_1(standalone, replica_config, options):
         cadb = certs.CertDB(realm_name, subject_base=subject_base)
         dsdb = certs.CertDB(realm_name, nssdir=dirname, subject_base=subject_base)
         trust_flags = dict(reversed(cadb.list_certs()))
-        trust_chain = cadb.find_root_cert('ipaCert')[:-1]
-        for nickname in trust_chain[:-1]:
-            cert = cadb.get_cert_from_db(nickname, pem=False)
+        trust_chain = certstore.get_ca_certs(
+            api.Backend.ldap2, api.env.basedn, api.env.realm, True)
+        for cert, nickname, _trusted, _ext_key_usage in trust_chain[:-1]:
```

Comment:
=> look for ca-signing-cert in /etc/pki/pki-tomcat/alias and use certdb.py:get_ca_nickname() to get the nickname with which it should be stored.